asdf/tags/wcs/wcs.py

```diff
@@ -343,54 +273,54 @@ def assert_equal(cls, old, new):
         for old_frame, new_frame in zip(old.frames, new.frames):
             helpers.assert_tree_match(old_frame, new_frame)
 
-class ICRSCoord(AsdfType):
```

Comment:
This conflicts with the latest frame generalisation changes in astropy/astropy#7079, in that asdf will use this and not the `BaseCoordinateFrame` schema in astropy. I don't know how to fix that other than remove it from here?

Reply:
Ideally I'd like to keep this tag in asdf for backwards compatibility, although it's not critical in this case.

A possible (but maybe not ideal) solution would be to make a special case of `ICRSCoord` and explicitly provide an implementation in Astropy as well. This should guarantee that it gets overridden.

Otherwise, we might need to provide some kind of API that allows existing tags to be overridden by the kind of subclass implementation you are working on in Astropy. But I suspect the logic necessary to support that might get kind of hairy.

Reply:
Can we just drop it for 2.0 😆